modules/dr-restoring-cluster-state.adoc

```diff
@@ -256,7 +256,7 @@ $ oc adm certificate approve <csr_name>
 +
 [source,terminal]
 ----
-$ sudo crictl ps | grep etcd | grep -v operator
+$ sudo crictl ps | grep etcd | grep -v operator | grep -v guard
```

Comment:
would the below do? As we are using the same https://github.com/openshift/openshift-docs/pull/55879/files

` sudo crictl ps | grep etcd | egrep -v "operator|etcd-guard" `

Reply:
Will adjust!